### PR TITLE
Bug 561435 API revision: requirements resolution: test coverage

### PR DESCRIPTION
**Pull request by eparovyshnaya**
-`BundleRequirements` service test set is complete

Signed-off-by: elena.parovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/i18n/EquinoxMessages.properties [hunk 1]
```diff
@@ -20,9 +20,9 @@ EquinoxRestrictions_title=Issues with licensing
 
 ComponentConfigurationResolver_error_invalid_bundle_context=Unable to extract configuration requirements: invalid BundleContext
 ComponentConfigurationResolver_error_invalid_component_rt=Unable to extract configuration requirements: invalid ServiceComponentRuntime
-ComponentRequirements_error_no_resource=Unable to extract configuration requirements: invalid {1}
-ComponentRequirements_requirement_for_resource={1} for {2} OSGi-component
-BundleRequirements_no_context=Bundle context for {1} OSGi-component
+ComponentRequirements_error_no_resource=Unable to extract configuration requirements: invalid {0}
+ComponentRequirements_requirement_for_resource={0} for {1} OSGi-component
+BundleRequirements_no_context=Bundle context for {0 OSGi-component
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/i18n/EquinoxMessages.properties:25]: Please complete the expression

File: bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleVendor.java [hunk 2]
```diff
@@ -33,7 +34,8 @@ protected BundleVendor(Bundle bundle) {
 
 	@Override
 	public String get() {
-		return bundle.getHeaders().get(Constants.BUNDLE_VENDOR);
+		return Optional.ofNullable(bundle.getHeaders().get(Constants.BUNDLE_VENDOR))//
+				.orElse("Unknown vendor"); //$NON-NLS-1$
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleVendor.java:38]: This should be visible for user and i18n is required

File: bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/requirements/RequirementFromCapability.java [hunk 3]
```diff
@@ -60,6 +60,7 @@ public Requirement get() {
 	}
 
 	private Requirement requirementFromAttributes(String feature, Map<String, Object> attributes) {
+		System.out.println(">>" + attributes); //$NON-NLS-1$
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/requirements/RequirementFromCapability.java:63]: Should be removed

File: bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleVendor.java (hunk 2 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleVendor.java:38]: Done

File: bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/requirements/RequirementFromCapability.java (hunk 3 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/requirements/RequirementFromCapability.java:63]: Done

File: bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/i18n/EquinoxMessages.properties (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/i18n/EquinoxMessages.properties:25]: Fixed

**Pull request merged**